app/boards/shields/qaz/qaz-layouts.dtsi

```diff
@@ -0,0 +1,186 @@
+            , <&key_physical_attrs 100 100  925  300       0     0     0>
+            ;
+    };
+
+    big_bar_layout: big_bar_layout {
+        compatible = "zmk,physical-layout";
+        display-name = "Big Bar";
+        transform = <&big_bar_transform>;
+        kscan = <&kscan0>;
+        keys //                      w   h    x    y     rot    rx    ry
+            = <&key_physical_attrs 100 100    0    0       0     0     0>
+            , <&key_physical_attrs 100 100  100    0       0     0     0>
+            , <&key_physical_attrs 100 100  200    0       0     0     0>
+            , <&key_physical_attrs 100 100  300    0       0     0     0>
+            , <&key_physical_attrs 100 100  400    0       0     0     0>
+            , <&key_physical_attrs 100 100  500    0       0     0     0>
+            , <&key_physical_attrs 100 100  600    0       0     0     0>
+            , <&key_physical_attrs 100 100  700    0       0     0     0>
+            , <&key_physical_attrs 100 100  800    0       0     0     0>
+            , <&key_physical_attrs 125 100  900    0       0     0     0>
+
+            , <&key_physical_attrs 125 100    0  100       0     0     0>
+            , <&key_physical_attrs 100 100  125  100       0     0     0>
+            , <&key_physical_attrs 100 100  225  100       0     0     0>
+            , <&key_physical_attrs 100 100  325  100       0     0     0>
+            , <&key_physical_attrs 100 100  425  100       0     0     0>
+            , <&key_physical_attrs 100 100  525  100       0     0     0>
+            , <&key_physical_attrs 100 100  625  100       0     0     0>
+            , <&key_physical_attrs 100 100  725  100       0     0     0>
+            , <&key_physical_attrs 100 100  825  100       0     0     0>
+            , <&key_physical_attrs 100 100  925  100       0     0     0>
+
+            , <&key_physical_attrs 175 100    0  200       0     0     0>
+            , <&key_physical_attrs 100 100  175  200       0     0     0>
+            , <&key_physical_attrs 100 100  275  200       0     0     0>
+            , <&key_physical_attrs 100 100  375  200       0     0     0>
+            , <&key_physical_attrs 100 100  475  200       0     0     0>
+            , <&key_physical_attrs 100 100  575  200       0     0     0>
+            , <&key_physical_attrs 100 100  675  200       0     0     0>
+            , <&key_physical_attrs 100 100  775  200       0     0     0>
+            , <&key_physical_attrs 150 100  875  200       0     0     0>
+
+            , <&key_physical_attrs 100 100    0  300       0     0     0>
+            , <&key_physical_attrs 100 100  100  300       0     0     0>
+            , <&key_physical_attrs 625 100  200  300       0     0     0>
+            , <&key_physical_attrs 100 100  825  300       0     0     0>
+            , <&key_physical_attrs 100 100  925  300       0     0     0>
+            ;
+    };
+
+    position_map {
```

Comment:
I think this position map can be improved by reducing the max position index and mapping the big bars to each other. Thanks to https://zmk-layout-helper.netlify.app/, I attempted to do that and reduced position count to 37, does this look better to you as well?

```dts
    position_map {
        compatible = "zmk,physical-layout-position-map";
        complete;

        split_bar_posmap {
            physical-layout = <&split_bar_layout>;
            positions
                = < 0  1  2  3  4  5  6  7  8  9>
                , <10 11 12 13 14 15 16 17 18 19>
                , <20 21 22 23 24 25 26 27    28>
                , <29 30 31   32    33  34 35 36>
                ;
        };

        split_big_bar_posmap {
            physical-layout = <&split_big_bar_layout>;
            positions
                = < 0  1  2  3  4  5  6  7  8  9>
                , <10 11 12 13 14 15 16 17 18 19>
                , <20 21 22 23 24 25 26 27    28>
                , <29 30 35   31    32  36 33 34>
                ;
        };

        big_bar_posmap {
            physical-layout = <&big_bar_layout>;
            positions
                = < 0  1  2  3  4  5  6  7  8  9>
                , <10 11 12 13 14 15 16 17 18 19>
                , <20 21 22 23 24 25 26 27    28>
                , <29 30 35   31    34  36 32 33>
                ;
        };
    };
```

Reply:
I do like it, but I think it would be nice to keep the two right side bottom row keys static, since they are present in all layouts.

Reply:
it's also worth noting that the split big bar layout doesn't use either of the split space bar positions, but the adjacent 1u key positions.

Reply:
> I do like it, but I think it would be nice to keep the two right side bottom row keys static, since they are present in all layouts.

This should be keeping them static, at least according to the helper. It also threw me off but this looks correct: e.g. when you switch from split bar to split big bar, position 36 (last item in posmap) in former maps to position 34 (last item in posmap) in latter.

Reply:
I took a stab at reconfiguring the space bar to be relatively consistent while maintaining key positions that are present in all layouts.

Reply:
Best would be to try it in Studio and confirm but I am reasonably sure that the helper works correctly.

Reply:
okay yeah the visualizer helped a lot, I was not understanding it. I associated the right split bars with the big bar while also keeping static buttons static.